packages/manager/modules/sign-up/src/form/activity/activity.controller.js

```diff
@@ -0,0 +1 @@
+export default class SignUpActivityCtrl {}
```

Comment:
If controller is empty, the `component` should probably only be declared with a `template`